### PR TITLE
TST: Test mask/where inplace don't modify views with CoW

### PR DESCRIPTION
**Pull request by lithomas1**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/copy_view/test_replace.py [hunk 1]
```diff
@@ -36,3 +37,22 @@ def test_replace_inplace_reference(using_copy_on_write):
...
     else:
         assert np.shares_memory(get_array(df, "a"), arr_a)
+
+
+@pytest.mark.parametrize("method", ["where", "mask"])
+def test_masking_inplace(using_copy_on_write, method):
+    df = DataFrame({"a": [1.5, 2, 3]})
+    df_orig = df.copy()
+    arr_a = get_array(df, "a")
+    view = df[:]  # noqa
```
Comment (phofl) [inline pandas/tests/copy_view/test_replace.py:47]: You can remove this (might have forgotten this as well), you are using view again a bit further down

Comment (lithomas1) [reply pandas/tests/copy_view/test_replace.py]: Thanks for the catch, updated your tests as well.

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
Failure unrelated

**Comment by phofl** [comment]
thx @lithomas1

**Pull request merged**